pkg/apis/veleroplugin/v1/upload.go

```diff
@@ -31,6 +31,10 @@ type UploadSpec struct {
 
 	// UploadCancel indicates request to cancel ongoing upload.
 	UploadCancel bool `json:"uploadCancel,omitempty"`
+
+	// BackupRepository provides backup repository info for upload. Used for
+	// multiple backup repository.
+	BackupRepository string `json:"backupRepository,omitempty"`
```

Comment:
Please rename BackupRepository to BackupRepositoryName.

Reply:
Changed to BackupRepositoryName